Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -45,15 +51,24 @@ public function configureFormFields(FormMapper $formMapper)
             'required' => false,
         );
 
-        $formMapper->with($this->formGroup, array(
-            'translation_domain' => 'CmfCoreBundle',
-            ))
-            ->add('publish_start_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_start_date',
-            ))
-            ->add('publish_end_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_end_date',
+        $formMapper
+            ->with($this->formTab, array_merge(
+                array('tab' => true), // use option instead of method to be compatible with Sonata <2.3
+                'form.tab_publish' === $this->formTab
+                    ? array('translation_domain' => 'CmfCoreBundle')
+                    : array()
```

Comment:
Each group can have its own translation domain (and each field too), which then overrides the inherited one from the tab.

The reason I did this: I've moved the publisable field to the general tab. I did that using:

````yaml
cmf_core:
    sonata_admin:
        extensions:
            publishable:
                form_tab: form.tab_general
                form_group: group_content
````

Leaving the translation domain in there would mean it'll override the translation domain of group_content and form.tab_general, causing issues as both sources are not available in the CmfCoreBundle translation domain.

So I did it this way:

* When using the default, it's the CoreBundle's responsibility to have a good translation
* When putting them in any other tab/group, it's the responsibility of the admin that created that tab/group to have good translation.